app/lib/concerns/bigquery_data_conversion.rb

```diff
@@ -0,0 +1,18 @@
+module BigqueryDataConversion
+private
+
+  def entity_data(attributes, table_name)
```

Comment:
just to confirm this has been tested across the different entities that extend it?

Reply:
Yes. The existing specs as well as the specs introduced to tes entity_event already cover this.